src/search/headerBar.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from 'react';
+import {
+  View,
+  SafeAreaView,
+  TextInput,
+  Text,
+  TouchableHighlight,
+} from 'react-native';
+import { useDispatch } from 'react-redux';
+import { useNavigation } from 'react-navigation-hooks';
+import styles from './styles';
+import { searchData } from '../redux/search/actionCreater';
+
+export default function SearchBar() {
+  const [value, onChangeText] = useState('');
```

Comment:
value is not enough descriptive, especially if you dont use 'value' in the setter method at all.